wiki/People/Beatmap_Nominators/id.md

```diff
@@ -0,0 +1,172 @@
+
+Menjadi seorang Beatmap Nominator
+----------------------------
+
+Dengan perombakan team pada Agustus 2014 lalu, bergabung menjadi Beatmap Nominator jauh lebih mudah dibandingkan dengan yang dulu. [Quality Assurance Team](https://osu.ppy.sh/g/7) akan mengumumkan pendaftaran Beatmap Nominator berupa post di [Pending Beatmaps Subforum](https://osu.ppy.sh/forum/6) dalam kurun waktu 1 atau 2 bulan.
+
+### Kriteria Dasar
+
+Modder yang ingin menjadi Beatmap Nominator harus memenuhi kriteria berikut:
++ Harus aktif dalam modding setidaknya kurang lebih 3 bulan jauh dari pengumuman.
++ Mempunyai kurang lebih 50 [Kudosu](/wiki/Glossary/#kudosu) dari total yang sudah diakumulasi.
++ Memberikan hasil modding setidaknya 2 atau 4 mod agar memenuhi syarat untuk menjadi bagian dari Beatmap Nominator.
+
+### Apa yang Berkualitas sebagai Mod Post
+
++ Setelah mendaftar, Anda akan diminta untuk memberikan jumlah total Kudosu serta jumlah postingan mod dalam kurun waktu 3 bulan terakhir, diurutkan berdasarkan bulan setelah mendaftar. Jadi jika pengumuman diposting pada bulan April, Anda akan melampirkan posting mod bulan Januari, Februari, dan Maret akan relevan. Bulan-bulan yang relevan juga akan terdaftar pada formulir submission.
++ Sebuah mod post akan memenuhi syarat apabila sudah menerima Kudosu dan dibuat dalam rentang waktu yang sudah tercantum dalam formulir submissions.
++ Sejak November 2017, modding beatmap telah dirubah menggunakan Modding V2 yang memiliki sistem reward berdasarkan upvote/downvote. Oleh karena itu nilai "Mod Post" sangat diperlukan dalam diskusi beatmap, yang dimana setidaknya satu masalah/saran mendapatkan reward upvote.
++ Gamemode (mode) apapun yang Anda Modding tidak menjadi masalah, karena aktivitas yang diukur adalah aktivitas keseluruhan di dalam komunitas Modding tersebut, dan bukan dari Gamemode tertentu.
++ Time dalam Mod Post yang telah diberikan Kudosu tidak masalah. Yang sangat penting dalam Mod Post ialah postingan date dan time dari UTC+0 yang membuktikan bahwa sudah diberikan Kudosu sesuai fakta.
+
+### Bagaimana Skor Minimal di Kalkulasi
+
++ Hal ini tergantung pada `Kudosu Score` dan total `Mod Score` pendaftar yang telah didapatkan selama bulan terakhir. Rumus Matematika diperlukan untuk menentukan nilai tersebut.
++ `Kudosu Score` ditentukan sesuai rumus sebagai berikut:
+
+![Rumus Kudosu Score](/wiki/shared/KDSCORE.png "Rumus Kudosu Score")
+
++ `Mod Score` per bulan ditentukan sesuai rumus sebagai berikut:
+
+![Rumus Mod Score](/wiki/shared/MODSCORE.png "Rumus Mod Score")
+
++ [Online Tool](http://modscore.little.moe/) ini dapat digunakan untuk mengkalkulasi total Mod Score.
++ Untuk menentukan `Required Minimum Score`, pendaftar harus menjelaskan/meringkas total `Kudosu Score` dan `Mod Score` mereka dari bulan pertama, dua, dan tiga.
```

Comment:
`meringkas` is redundant, too literal, and less accurate compared to `menjelaskan`, so might as well remove it. Might want to change `dari` to `pada`, and add a `ke-` prefix to `dua` and `tiga` as well.